commands/miner.go

```diff
@@ -42,14 +42,15 @@ type MinerCreateResult struct {
 
 var minerCreateCmd = &cmds.Command{
 	Helptext: cmdkit.HelpText{
-		Tagline: "Create a new file miner with <pledge> sectors and <collateral> FIL",
+		Tagline: "Create a new file miner with <collateral> FIL",
 		ShortDescription: `Issues a new message to the network to create the miner, then waits for the
 message to be mined as this is required to return the address of the new miner.
-Collateral must be greater than 0.001 FIL per pledged sector.`,
+Collateral will be committed at the rate of 0.001FIL per sector. When the 
```

Comment:
Where does this 0.001 come from? Is this from the network collateral cost function, which happens to be hardcoded right now? I think it's going to rot very quickly: it depends on sector size and (probably) block height.

Reply:
@anorth 

> Where does this 0.001 come from?

The `0.001` value comes from [the `MinimumCollateralPerSector` constant](https://github.com/filecoin-project/go-filecoin/blob/master/actor/builtin/miner/miner.go#L37).

> I think it's going to rot very quickly: it depends on sector size and (probably) block height.

It will rot - yes. Unsure how long it will take to rot, though. I do not know how far into the future it will be when we implement a real `CollateralForSector`.

If you prefer, I can remove the "Collateral will be committed at the rate of 0.001FIL per sector" message. If that message is gone, how best can we communicate to miners the relationship between the FIL they use as collateral when creating a miner and the number of sectors that miner will be able to commit?

Reply:
Ok, let's leave it until a new command invokes CollateralForSector to report it (regardless of whether the returned value is "real").